.gitignore

```diff
@@ -1,3 +1,3 @@
-.idea/*
-*.iml
-target/*
+.idea
+app.log
```

Comment:
in the config file, we wrote logs/app.log path. Should we change the path to the log file here?